docs/filters.rst

```diff
@@ -0,0 +1,87 @@
+.. _filters:
+
+Filtering
+=========
+
+REST resource filtering is provided via method ``_filter_queryset``. If you want implement your own filter technique
+you can add here your own code. Input params of the method are only ``qs`` which represents data that you want to filter
+(for model resources it is django QuerySet object).
+
+Better way is used pre-implemented managers defined in model ``BaseModelResource``::
+
+    class BaseModelResource(DefaultRESTModelResource, BaseObjectResource):
+        filter_manager = MultipleFilterManager()
+
+        def _filter_queryset(self, qs):
+            if self.filter_manager:
+                return self.filter_manager.filter(self, qs, self.request)
+            else:
+                return qs
+
+There method ``_filter_queryset`` uses manager to filter output data. You can change manager with filter_manager
+attribute. If you set filter_manager to ``None``. Filtering will be disabled. Default filter manager is
+``MultipleFilterManager``.
```

Comment:
In the BaseModelResource, ``_filter_queryset`` method uses a manager to filter the output data.
You can override the default manager with the ``filter_manager`` attribute.
To disable filtering, set ``filter_manager = None``.
The default filter manager is ``MultipleFilterManager``.